engineering-plans/0002-ethereum-tracing-cache.md

```diff
@@ -0,0 +1,122 @@
+# PLAN-0002: Ethereum Tracing Cache
+
+<dl>
+  <dt>Author</dt>
+  <dd>Zachary Burns</dd>
+
+  <dt>Implements</dt>
+  <dd><a href="../rfcs/0002-ethereum-tracing-cache.md">RFC-0002 Ethereum Tracing Cache</a></dd>
+
+  <dt>Engineering Plan pull request</dt>
+  <dd><a href="https://github.com/graphprotocol/rfcs/pull/9">https://github.com/graphprotocol/rfcs/pull/9</a></dd>
+
+  <dt>Obsoletes (if applicable)</dt>
+  <dd>None</dd>
+
+  <dt>Date of submission</dt>
+  <dd>2019-12-20</dd>
+
+  <dt>Date of approval</dt>
+  <dd>YYYY-MM-DD</dd>
+
+  <dt>Approved by</dt>
+  <dd>First Person, Second Person</dd>
+</dl>
+
+## Summary
+
+Implements RFC-0002: Ethereum Tracing Cache
+
+## Implementation
+
+These changes happen within `ethereum_adapter.rs`, `store.rs` and `db_schema.rs`.
+
+### ServiceCache trait
```

Comment:
Should we just call this `IndexingCache` or something? I'm not sure what the `Service` stands for. 😉

Reply:
> Should we just call this `IndexingCache` or something? I'm not sure what the `Service` stands for. 😉

`Service` stands for service in the micro-service sense.  We are making some remote call that needs to be cached. I'm not sure what `Indexing` stands for. But, I'm amenable to whatever name.